### ISSUE TITLE
Issue with "WAI-ARIA Roles": no menu role

### ISSUE THREAD

**Issue by Guy-Moses** [mdn/content#535]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

#### What information was incorrect, unhelpful, or incomplete?
The ARIA 1.1 [menu role](https://www.w3.org/TR/wai-aria-1.1/#menu) does not have an entry in this page.
Same goes for menu sub items: [menuitem](https://www.w3.org/TR/wai-aria-1.1/#menuitem), [menuitemradio](https://www.w3.org/TR/wai-aria-1.1/#menuitemradio) and [menuitemcheckbox](https://www.w3.org/TR/wai-aria-1.1/#menuitemcheckbox).

Additionally, looks like these roles don't have a dedicated page.

#### Specific section or headline?
n/a - no sections in this page

#### What did you expect to see?
Entries (heading, short description and a link to the entry page) for the four roles mentioned above, similar to other entries in this page.

Additionally, each of these four should have their own page, similar to other entries in this page.

#### Did you test this? If so, how?
n/a

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/accessibility/aria/roles/index.html
* Report started: 2020-12-21T19:23:50.547Z

</details>

**Comment by chrisdavidmills** [comment]
I'm going to close this one for now, as eventually we are intending to cover missing role reference docs in a more substantial piece of work; see our RFC proposal at https://github.com/mdn/content/blob/main/rfcs/aria-roles.md
